core/services/vrf/mocks/aggregator_v3_interface.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.13.0-beta.1. DO NOT EDIT.
+// Code generated by mockery v2.12.2. DO NOT EDIT.
```

Comment:
I think you should probably revert the changes on these files, since you're using an older version of mockery.